packages/datadog-plugin-jest/src/index.js

```diff
@@ -237,6 +237,7 @@ class JestPlugin extends Plugin {
         testSuiteSpan.setTag('error', new Error(errorMessage))
       }
       testSuiteSpan.finish()
+      finishAllTraceSpans(testSuiteSpan)
```

Comment:
this is important because suites run in a different process than the session, so calling `finishAllTraceSpans` only on the session span is not enough

Reply:
Can you add this comment in the code instead?

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/2431/commits/72f64280463d34fe89af2d5e65fe109d42df69a6